v2/AdminCore.Common/Interfaces/IDashboardService.cs

```diff
@@ -9,7 +9,7 @@ public interface IDashboardService
   {
     IList<EventDto> GetEmployeeDashboardEvents(int employeeId, DateTime date);
 
-    IList<EventDto> GetEmployeeEvents(int employeeId, DateTime date);
+    IList<EventDto> GetEmployeeEventsForAGivenMonth(int employeeId, DateTime date);
```

Comment:
I'm not so sure about these naming conventions, perhaps a bit too long winded. Can we do something like 'GetEmployeeEventsForMonth'

Reply:
Changed to GetEmployeeEventsForMonth